.github/workflows/node-increase-disk.yml

```diff
@@ -0,0 +1,49 @@
+name: Increase Node Disk
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
```

Comment:
Where is this variable used? And have you considered making it an [enum](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/)?

Reply:
it's one of the things I need to validate, but I think by using `type: environment` I can inherit the right AWS credentials, etc. from the repository environment with the matching name